docs/SDL/GetStatusUpdate/index.md

```diff
@@ -11,23 +11,31 @@ Purpose
 
 ### Request
 
+In case HMI needs to find out current status of PTU and sends _GetStatusUpdate_request_ to SDL, it must respond with the current update status code to HMI.
+The request _GetStatusUpdate_ duplicates the functionality of the notification _OnStatusUpdate_. In case the policy update status is being changed.(e.g. an update is finished successfully or retry strategy failed), SDL must send notification _OnStatusUpdate_ to HMI with the corresponding UpdateStatus code, whereas _GetStatusUpdate_ allowes to request the status of policy table at any time, not on update only.
+
+!!! MUST:
+Send a request to SDL if it needs to get a current policy update status according to its workflows.
+!!!
+
 #### Parameters
 
-|Name|Type|Mandatory|Additional|
-|:---|:---|:--------|:---------|
+This RPC has no additional parameter requirements.
 
 ### Response
 
 #### Parameters
 
-|Name|Type|Mandatory|Additional|
-|:---|:---|:--------|:---------|
-|status|[Common.UpdateResult](../../common/enums/#updateresult)|true||
+|Name|Type|Mandatory|
+|:---|:---|:--------|
+|status|[Common.UpdateResult]|true|
+
+[Common.UpdateResult]: ../../common/enums/#updateresult
 
 ### Sequence Diagrams
 |||
 GetStatusUpdate
-![GetStatusUpdate](./assets/GetStatusUpdate.jpg)
+![GetStatusUpdate](./assets/getstatusupdate.jpg)
```

Comment:
Sorry for the confusion, but the actual file names need to have the same case as they are in the repository. If you look in [the preview](https://smartdevicelink.com/en/docs/pull_request/6f78b8bcbb3501df5a8cf1da744f11c5e79456c0/sdl/getstatusupdate/), you will see that this link is broken by this change.

1. All directories should be in lowercase (this is just how the routes are set up).
2. All image file names should match the case in this repository.